Library/Extensions/StringExtensions.cs

```diff
@@ -69,6 +69,7 @@ public static string ToPascalCase(this string source)
             if (source.IsNullOrEmpty()) return source;
 
             source = source.Replace("_", " "); // so we know where the word breaks are
+            source = source.Replace("'", ""); // e.g. Diner's Club (API should really be returning 'Diners Club')
```

Comment:
Thanks for pointing this out! This will be fixed soon and before this PR is merged in, so lets :cut: